package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "@distube/ytdl-core",
-  "description": "DisTube fork of ytdl-core. YouTube video downloader in pure javascript.",
```

Comment:
Please revert your changes in package.json

Reply:
Ready